addons/binding/org.openhab.binding.sleepiq/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sleepiq"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- SleepIQ Cloud Service -->
+    <bridge-type id="cloud">
+        <label>SleepIQ Cloud</label>
+        <description>The SleepIQ cloud service provides connections to all beds in an account.</description>
+
+        <properties>
+            <property name="vendor">Select Comfort</property>
+        </properties>
+        
+        <config-description>
+            <parameter name="username" type="text">
+                <label>Username</label>
+                <description>Username of a registered SleepIQ account owner</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="password" type="text">
+                <context>password</context>
+                <label>Password</label>
+                <description>Password of a registered SleepIQ account owner</description>
+                <required>false</required>
+            </parameter>
+            <parameter name="pollingInterval" type="integer" min="30" step="30">
+                <label>Polling Interval</label>
+                <description>Seconds between fetching values from the cloud service</description>
+                <required>true</required>
+                <default>60</default>
+            </parameter>
+            <parameter name="logging" type="boolean">
+                <label>Logging flag</label>
+                <description>Enables or disables logging in the SleepIQ API</description>
+                <required>false</required>
+                <default>false</default>
+            </parameter>
+        </config-description>
+    </bridge-type>
+    
+    <!-- Bed thing with a single chamber -->
+    <!-- Single chamber bed support is disabled until test devices are available
```

Comment:
My rationale for including the reference was to give myself (or someone else) a starting point for adding single-chamber bed functionality and to document that the hardware exists even though it is not currently supported by the binding. If this is not in line with project norms, let me know and I will remove it.